training-node/app/controllers/user.js

```diff
@@ -0,0 +1,40 @@
+const userServices = require('../services/user'),
+  logger = require('../logger'),
+  errors = require('../errors'),
+  bcrypt = require('bcrypt');
+
+const validateUser = user =>
+  new Promise((resolve, reject) => {
```

Comment:
I dont like using promises for non asynchronic code. This method can simply return an object with `isValid` and `message` keys